gradio/routes.py

```diff
@@ -728,124 +635,14 @@ def get_types(cls_set: List[Type]):
-            setattr(self, key, value)
-
-    def __getitem__(self, item):
-        return self.__dict__[item]
-
-    def __setitem__(self, item, value):
-        self.__dict__[item] = value
-
-    def __iter__(self):
-        for key, value in self.__dict__.items():
-            if isinstance(value, Obj):
-                yield (key, dict(value))
-            else:
-                yield (key, value)
-
-    def __contains__(self, item) -> bool:
-        if item in self.__dict__:
-            return True
-        for value in self.__dict__.values():
-            if isinstance(value, Obj) and item in value:
-                return True
-        return False
-
-    def keys(self):
-        return self.__dict__.keys()
-
-    def values(self):
-        return self.__dict__.values()
-
-    def items(self):
-        return self.__dict__.items()
-
-    def __str__(self) -> str:
-        return str(self.__dict__)
-
-    def __repr__(self) -> str:
-        return str(self.__dict__)
-
-
-@document()
-class Request:
-    """
-    A Gradio request object that can be used to access the request headers, cookies,
-    query parameters and other information about the request from within the prediction
-    function. The class is a thin wrapper around the fastapi.Request class. Attributes
-    of this class include: `headers`, `client`, `query_params`, and `path_params`. If
-    auth is enabled, the `username` attribute can be used to get the logged in user.
-    Example:
-        import gradio as gr
-        def echo(name, request: gr.Request):
-            print("Request headers dictionary:", request.headers)
-            print("IP address:", request.client.host)
-            return name
-        io = gr.Interface(echo, "textbox", "textbox").launch()
-    """
-
-    def __init__(
-        self,
-        request: fastapi.Request | None = None,
-        username: str | None = None,
-        **kwargs,
-    ):
-        """
-        Can be instantiated with either a fastapi.Request or by manually passing in
-        attributes (needed for websocket-based queueing).
-        Parameters:
-            request: A fastapi.Request
-        """
-        self.request = request
-        self.username = username
-        self.kwargs: Dict = kwargs
-
-    def dict_to_obj(self, d):
-        if isinstance(d, dict):
-            return json.loads(json.dumps(d), object_hook=Obj)
-        else:
-            return d
-
-    def __getattr__(self, name):
-        if self.request:
-            return self.dict_to_obj(getattr(self.request, name))
-        else:
-            try:
-                obj = self.kwargs[name]
-            except KeyError as ke:
-                raise AttributeError(
-                    f"'Request' object has no attribute '{name}'"
-                ) from ke
-            return self.dict_to_obj(obj)
-
-
 @document()
 def mount_gradio_app(
     app: fastapi.FastAPI,
     blocks: gradio.Blocks,
     path: str,
-    gradio_api_url: str | None = None,
```

Comment:
Is this breaking change OK?

Reply:
I reverted it because this made the CI fail, maybe at building the document.
https://github.com/gradio-app/gradio/actions/runs/6000343454/job/16272211763?pr=5165